sdc/crypto/test/test_jwt_helper.py

```diff
@@ -276,3 +289,29 @@ def assert_in_decode_signed_jwt_exception(self, jwe, error):
         if error not in ite.exception.value:
             raise AssertionError(
                 '"{}" not found in decode exception. Actual exception message [{}]'.format(error, ite.exception.value))
+
+    def test_encode_with_dict_and_string(self):
+        payload = {
+            'answers': [
+                {
+                    'first_name': 'Bob',
+                    'last_name': 'Smith',
+                    'group_instance': 0,
+                    'age': 50,
+                    'answer_id': 'personal_details'
+                }
+            ]
+        }
+        claims = json.dumps(payload)
```

Comment:
- I think it would be clearer if we renamed `payload` to `claims_as dict` and `claims` to `claims_as_string`
- The payload should be the simplest it can be to demonstrate the functionality being tested. Currently there are three string fields and two number fields. We should change this to remove the duplicated ones and test more types, something like:
```
claims_as_dict = {
  'data': [
    {
      'string': 'something',
      'boolean': true,
      'number': 10,
      'decimal': 10.1
      'null': null
    }
  ]
}
```

Reply:
@ajmaddaford agreed will make changes